README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Build Status](https://travis-ci.org/terriann/CsvToPlainTextTable.svg?branch=master)](https://travis-ci.org/terriann/CsvToPlainTextTable)
 [![Coverage Status](https://coveralls.io/repos/github/terriann/CsvToPlainTextTable/badge.svg?branch=code-coverage)](https://coveralls.io/github/terriann/CsvToPlainTextTable?branch=code-coverage)
+[![Package Security Status with David](https://david-dm.org/terriann/CsvToPlainTextTable.svg)](https://david-dm.org/)
```

Comment:
This should not link to the David home page, update to https://david-dm.org/terriann/CsvToPlainTextTable